Assets/Scripts/Capabilities/Move.cs

```diff
@@ -43,6 +45,9 @@ private void FixedUpdate()
 
         body.velocity = velocity;
 
+        isMoving = Mathf.Abs(velocity.x) > 0 ? true : false;
```

Comment:
nit (minor): I've seen tutorials recommend using [Mathf.Epsilon](https://docs.unity3d.com/ScriptReference/Mathf.Epsilon.html) instead of `0`, because sometimes a value can be verrrry close to zero but not zero.

Reply:
That makes sense, I'll update that